### PR TITLE
Replaces tool sprites with Dromkis

### PR DESCRIPTION
**Pull request by Dorodomki**
## About The Pull Request

adds the keyring to the masons so they can carry their keys a bit better but won't get any key on it and got to add it manually, the blacksmith towner gets the peasant knife since i forgot to add it, some tools get updated to use the proper sprites, i used the wrong paths and now is fixed

## Why It's Good For The Game

fixes an oopsie and adds more QOL

## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

**Review by Optimism333 — COMMENTED** [review]
Please speak with pig about making these changes compatible with #902 when you have a chance.

Once that is done, I'll happily approve this.

**Comment by Optimism333** [comment]
image

Remember, keep formatting consistent.

Should be written like this: 
- /obj/item/rogueweapon/huntingknife/villager = 1
- /obj/item/flashlight/flare/torch/lantern = 1
- /obj/item/keyring = 1

File: code/modules/jobs/job_types/roguetown/serfs/mason.dm [hunk 1]
```diff
@@ -55,7 +55,7 @@
 	beltl = /obj/item/rogueweapon/hammer/claw
 	backl = /obj/item/storage/backpack/rogue/backpack
 	id = /obj/item/clothing/ring/silver/makers_guild
-	backpack_contents = list(/obj/item/roguekey/mason = 1, /obj/item/flint = 1, /obj/item/flashlight/flare/torch/lantern, /obj/item/rogueweapon/huntingknife/villager)
+	backpack_contents = list(/obj/item/roguekey/mason = 1, /obj/item/flint = 1, /obj/item/flashlight/flare/torch/lantern, /obj/item/rogueweapon/huntingknife/villager, /obj/item/keyring)
```
Comment (NPC1314) [inline code/modules/jobs/job_types/roguetown/serfs/mason.dm:58]: add = 1 to lantern and knife and keyring to safe it up

File: code/modules/roguetown/roguejobs/gravedigger/tools.dm [hunk 2]
```diff
@@ -6,8 +6,8 @@
 	name = "shovel"
 	desc = ""
 	icon_state = "shovel"
-	icon = 'modular/Neu_Farming/icons/farmtools.dmi'
-//	icon = 'icons/roguetown/weapons/64.dmi'
+//icon = 'modular/Neu_Farming/icons/farmtools.dmi'
+	icon = 'icons/roguetown/weapons/tools.dmi'
```
Comment (NPC1314) [inline code/modules/roguetown/roguejobs/gravedigger/tools.dm:10]: changes in the tools.dm not desired at this point as split up edits,  make a proper shovel PR later with screenshots before and after etc and we can revisit the topic

File: code/modules/jobs/job_types/roguetown/serfs/mason.dm (hunk 1 above)
Comment (Dorodomki) [reply code/modules/jobs/job_types/roguetown/serfs/mason.dm:58]: weird, it works perfectly fine without the =1

File: code/modules/roguetown/roguejobs/gravedigger/tools.dm (hunk 2 above)
Comment (Dorodomki) [reply code/modules/roguetown/roguejobs/gravedigger/tools.dm:10]: then i change the old shovel DMI with a new DMI since i have to re add the spades too

**Comment by Optimism333** [comment]
I'd like to see what the new farmtools look like in the pr body, please.

After Pig gives a review, I'll give my two cents as well.

**Review by Optimism333 — COMMENTED** [review]
I'd like to see what the new farmtools look like in the pr body, please.

After Pig gives a review, I'll give my two cents as well.

**Comment by Dorodomki** [comment]
Captura de pantalla (2237)
Captura de pantalla (2238)
Captura de pantalla (2239)

**Review by Optimism333 — COMMENTED** [review]
I far prefer the neu farming sprites for the farm tools to be honest.

I don't think these farm tool sprites are improvements.

We can merge the fixes for the other sprites, but I don't want to replace our farm tools with these.

**Comment by NPC1314** [comment]
The new slimmed down version been voted on and its time we put it to the test to see how it looks ingame

**Review by NPC1314 — DISMISSED** [review]
Un-needed and of dubious quality but since thats what the sprite team wants and no one else cares enough, here you go.

Sprite replacment PRs are not going to be needed for the foreseeable future though probably.

**Review by NPC1314 — APPROVED** [review]

**Pull request merged**